src/test/java/plugins/WorkflowMultibranchTest.java

```diff
@@ -1,40 +1,51 @@
 package plugins;
 
+import java.io.IOException;
+import javax.mail.MessagingException;
 import org.apache.commons.io.IOUtils;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.anyOf;
+import static org.hamcrest.Matchers.not;
+import static org.jenkinsci.test.acceptance.Matchers.hasAction;
 import org.jenkinsci.test.acceptance.junit.AbstractJUnitTest;
 import org.jenkinsci.test.acceptance.junit.WithPlugins;
+import org.jenkinsci.test.acceptance.plugins.credentials.CredentialsPage;
+import org.jenkinsci.test.acceptance.plugins.credentials.ManagedCredentials;
+import org.jenkinsci.test.acceptance.plugins.credentials.UserPwdCredential;
 import org.jenkinsci.test.acceptance.plugins.maven.MavenInstallation;
 import org.jenkinsci.test.acceptance.plugins.workflow_multibranch.GithubBranchSource;
 import org.jenkinsci.test.acceptance.po.Build;
 import org.jenkinsci.test.acceptance.po.WorkflowJob;
 import org.jenkinsci.test.acceptance.po.WorkflowMultiBranchJob;
 import org.junit.Assert;
+import static org.junit.Assert.assertEquals;
 import org.junit.Before;
 import org.junit.Test;
 
-import javax.mail.MessagingException;
-import java.io.IOException;
-
-import static org.hamcrest.CoreMatchers.containsString;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.anyOf;
-import static org.hamcrest.Matchers.not;
-import static org.jenkinsci.test.acceptance.Matchers.hasAction;
-import static org.junit.Assert.assertEquals;
-
 /**
  * Tests a multibranch pipeline flow
  */
-@WithPlugins({"git", "javadoc@1.4", "workflow-job", "workflow-cps", "workflow-basic-steps", "workflow-durable-task-step", "workflow-multibranch", "github-branch-source"})
+@WithPlugins({"git", "javadoc@1.4", "workflow-basic-steps", "workflow-durable-task-step", "workflow-multibranch", "github-branch-source"})
 public class WorkflowMultibranchTest extends AbstractJUnitTest {
 
+    private static final String GH_CRED_ID = "ghCred";
+
     @Before
     public void setup() {
         MavenInstallation.installMaven(jenkins, "M3", "3.3.9");
     }
 
     @Test
     public void testMultibranchPipeline() throws IOException, MessagingException {
+        final CredentialsPage mc = new CredentialsPage(jenkins, ManagedCredentials.DEFAULT_DOMAIN);
+        mc.open();
+        final UserPwdCredential cred = mc.add(UserPwdCredential.class);
+        cred.username.set("varyvoltest");
+        cred.password.set("ff6f95500a17b42872ef1d525e94cc1066aec488");
```

Comment:
@jglick I assume this credentials are being used in other tests, and you are using them here.